postgresql_metrics/prepare_db.py

```diff
@@ -149,6 +151,24 @@ def create_pgstattuples_extension(db_connection):
         c.execute("CREATE EXTENSION IF NOT EXISTS pgstattuple;")
         c.execute(sql)
 
+def check_if_incoming_replication_status_view_exists(db_connection):
+    with db_connection.cursor() as c:
+        c.execute("SELECT table_name FROM information_schema.tables "
+                  "WHERE table_name=%s", (INCOMING_REPLICATION_STATS_VIEW,))
+        result = c.fetchone()
+        return bool(result) and result[0] == INCOMING_REPLICATION_STATS_VIEW
+
+def create_incoming_replication_status_view(db_connection):
+    LOG.info("creating view {}", INCOMING_REPLICATION_STATS_VIEW)
+    func_sql = """CREATE OR REPLACE FUNCTION public.stat_incoming_replication()
```

Comment:
Function and the view has the same name. Is this intended?

Reply:
I've set it to the same to keep it consistent with what we have `REPLICATION_STATS_VIEW`, that defines both view and function as `public.pg_stat_repl`

Reply:
Good point. Totally ok to have it like this.